### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by NimRegev**
Updated with initial new list of questions and answers

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/nima-cap-faqs/docs/

File: _docs/getting-started/faq.md [hunk 1]
```diff
@@ -3,37 +3,37 @@ title: "Frequently asked questions"
...
+**Q. Which deployment environments does CSDP support?**  
 
-## Possible Installations
+A. The current release of CSDP supports hybrid deployment environments. Stay tuned for our announcement on support for SaaS and on-premises deployments.   
 
-**Q. what are the supported git providers**  
-A. Codefresh supports most of the git providers that exists including: Github, Gitlab, Bitbucket.
+**Q. How does CSDP relate to Open Source Argo?**   
 
+A. CSDP creates a conformed fork of the Argo Project on installation. You get an enterprise-supported version of the Agro Project comprising Argo Workflows, Argo Events, Argo CD, and Argo Rollouts. You can take advantage of what Argo Project offers with the extended functionality that CSDP brings to it.
```
Comment (itai-codefresh) [inline _docs/getting-started/faq.md:21]: the forks are maintained without relation to an installation

**Review by itai-codefresh — APPROVED** [review]

**Comment by NimRegev** [comment]
Updated with Itai's review comments

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/nima-cap-faqs/docs/

**Pull request merged**